packages/block-editor/src/components/list-view/branch.js

```diff
@@ -116,6 +117,16 @@ function ListViewBranch( props ) {
 
 	const { expandedState, draggedClientIds } = useListViewContext();
 
+	// For long lists where the selected item may fall outside of the current window,
+	// pass a reference to the corresponding placeholder row for the selected item.
+	// The "real" selected item is also observed in ListViewBlock, when rendered.
+	const invisibleSelectedItemRef = useRef();
+	useListViewScrollIntoView( {
```

Comment:
Is it possible to use the hook only here and remove the usage in `ListViewBlock`?

I find the naming `invisibleSelectedItemRef` a bit confusing, and it might be cleaner if there were only one `selectedBlockRef` that were passed to either the placeholder row or `ListViewBlock` as a prop.